lib/puppet/provider/templates/webadministration/json_1.7.ps1.erb

```diff
@@ -339,7 +339,7 @@ BEGIN {
   <xsl:template match="*[count(../*[name(../*)=name(.)])=count(../*) and count(../*)&gt;1]">
 -->
   <xsl:template match="*[count(../*[name(../*)=name(.)])=count(../*)]">
-    <xsl:variable name="inArray" select="translate(local-name(),'OBJECT','object')='object' or ../@Type[starts-with(.,'System.Collections') or contains(.,'[]') or (contains(.,'[') and contains(.,']'))]"/>
+    <xsl:variable name="inArray" select="local-name(),'OBJECT','object')='object' or ../@Type[starts-with(.,'System.Collections') or contains(.,'[]') or (contains(.,'[') and contains(.,']')]"/>
```

Comment:
pretty sure this has nothing to do with the translate module, and will break the functionality of the module.

Reply:
Yeh, wasn't really sure about this.
Will rip it out now.